lib/adapters/adapter.ex

```diff
@@ -78,7 +62,7 @@ defmodule Janus.Transport.WS.Adapter do
   @doc """
   Sends payload via given WebSocket.
   """
-  @callback send(websocket :: websocket_t(), payload :: payload_t()) :: :ok | {:error, any}
+  @callback send(payload :: payload_t(), websocket :: websocket_t()) :: :ok | {:error, any}
```

Comment:
1. websocket ->  state
2. modify the type (`websocket_t` does not have to be pid)